### ISSUE TITLE
ClientConfig authToken 声明问题

### ISSUE THREAD

**Issue by yudongdong-1** [sashabaranov/go-openai#786]
ClientConfig authToken 声明问题，导致无法传参

**Comment by Biskwit** [comment]
agree, need authToken => AuthToken to allow us using custom ClientConfig

**Comment by t0uh33d** [comment]
Yes, this needs to be fixed :(

**Comment by AlienegraGeek** [comment]
Agree, add plz

**Comment by litao09h** [comment]
config := openai.DefaultConfig(apiKey)
	config.BaseURL = "https://api.xxxxx.com"
	openAIClient := openai.NewClientWithConfig(config)
